cve_bin_tool/checkers/thunderbird.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for thunderbird
+
+https://www.cvedetails.com/product/3678/Mozilla-Thunderbird.html?vendor_id=452
+
+"""
+from __future__ import annotations
+
+from cve_bin_tool.checkers import Checker
+
+
+class ThunderbirdChecker(Checker):
+    CONTAINS_PATTERNS: list[str] = []
+    FILENAME_PATTERNS: list[str] = [r"thunderbird"]
+    VERSION_PATTERNS = [
+        r"ENCODER=Mozilla([0-9]+.[0-9]+(.[0-9]+)?)",
```

Comment:
This pattern exists in Mozilla Firefox checker too. This could probably lead to false positives (we also might want to take out the pattern from Firefox if this pattern is present in Thunderbird)

Reply:
The issue is with this pattern our tool will say the detected binary is Firefox if it scans Thunderbird (false positive). That's probably not what we want.